src/pretix/presale/templates/pretixpresale/event/order.html

```diff
@@ -373,7 +373,26 @@ <h3 class="panel-title">
                 {% if user_cancel_allowed %}
                     <li class="list-group-item">
                         {% if order.status == "p" and order.total != 0 %}
-                            {% if order.user_cancel_fee >= order.total %}
+                            {% if request.event.settings.cancel_allow_user_paid_require_approval and request.event.settings.cancel_allow_user_paid_require_approval_fee_unknown %}
+                                <p>
+                                    {% blocktrans trimmed %}
+                                        You can request to cancel this order.
+                                    {% endblocktrans %}
+                                    {% blocktrans trimmed %}
+                                        If your request is approved, the organizer will determine if you will receive a
+                                        full refund or if a cancellation fee is deducted according to their cancellation
+                                        policy.
+                                    {% endblocktrans %}
+                                    {% if request.event.settings.cancel_allow_user_paid_refund_as_giftcard == "force" %}
+                                        {% trans "The refund will be issued in form of a gift card that you can use for further purchases." %}
+                                    {% elif request.event.settings.cancel_allow_user_paid_refund_as_giftcard == "option" %}
+                                        {% trans "The refund can be issued to your original payment method or as a gift card." %}
+                                    {% elif request.event.settings.cancel_allow_user_paid_refund_as_giftcard != "manually" %}
```

Comment:
I can’t think of another option in these giftcard-settings, but would it be more future-proof to compare it to the actual value "off" for the of refund to original payment method?

```suggestion
                                    {% elif request.event.settings.cancel_allow_user_paid_refund_as_giftcard == "off" %}
```

Reply:
Not really part of this PR but why not ;)